shared/src/commonMain/kotlin/org/hyperskill/app/project_selection/presentation/ProjectSelectionListFeature.kt

```diff
@@ -79,6 +82,7 @@ object ProjectSelectionListFeature {
 
     internal sealed interface ContentFetchResult : Message {
         data class Success(
+            val profile: Profile,
```

Comment:
I think we can delete this, I don't see any further usages

Reply:
It is used in the reducer. See [ProjectSelectionListReducer](https://github.com/hyperskill/mobile-app/blob/08720628660312b42e40350e78c710b4ff4824cc/shared/src/commonMain/kotlin/org/hyperskill/app/project_selection/presentation/ProjectSelectionListReducer.kt#L149-L153)